docs/security-updates/vex/index.rst

```diff
@@ -22,10 +26,57 @@ VEX data for Ubuntu releases is available from three official sources:
++---------------+--------------------------+------------------------------------------------------------------------------------+---------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| needed        | Vulnerable               | CVE needs to be patched                                                            | affected            | action_statement: The package is vulnerable and needs fixing.                                                                                                                    |
++---------------+--------------------------+------------------------------------------------------------------------------------+---------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ignored       | Ignored                  | Canonical Security Engineering team or upstream has ignored this CVE               | affected            | action_statement: This package is vulnerable to the CVE, the problem is understood, but the Ubuntu Security Team decided to not fix it. CVE notes / justification: [...]         |
++---------------+--------------------------+------------------------------------------------------------------------------------+---------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| released      | Fixed                    | Canonical Security Engineering team has fixed the CVE and released the new version | fixed               | status_notes: A fix has been published. Additional notes provided if the fix is part of Ubuntu Pro (e.g., esm-infra, esm-apps).                                                  |
++---------------+--------------------------+------------------------------------------------------------------------------------+---------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+The *CVE notes / justification* information are provided by the Ubuntu CVE Tracker and are being actively maintained by the Ubuntu Security Team.
+
+When it comes to USN (Ubuntu Security Notices) VEX data, the documents will only state the CVE entries that were fixed as a result of the update. Thus, it is expected that every statement in a USN VEX document will have a **fixed** status.
+The *status_notes* will indicate if the update is widely available, or if an Ubuntu Pro subscription is required.
```

Comment:
```suggestion
The *status_notes* will indicate if the update is widely available, or if you need an 
Ubuntu Pro subscription to get the update.
```
I think this phrasing (or something like this) is a bit more clear how the Ubuntu Pro subscription relates.